apps/chat-e2e/src/tests/sharedFilesAttachments.test.ts

```diff
@@ -83,14 +81,15 @@ dialSharedWithMeTest(
     // let imageInFolderUrl2: string;
     let shareByLinkResponse: ShareByLinkResponseModel;
     let shareFolderByLinkResponse: ShareByLinkResponseModel;
-    let defaultModel: ModelIds;
+    let defaultModel: string;
     let conversationInFolder: Conversation;
     //TODO EPMRTC-4135 blocked by the #1076
     // let conversationToMove: Conversation;
     const folderName = 'Folder with conversation';
+    const appdataFolderName = 'appdata';
+    const imagesFolderName = 'images';
```

Comment:
I'd move `appdataFolderName`, `imagesFolderName` into `ExpectedConstants` because they may require for other tests as well

Reply:
done